### DISCUSSION TITLE
Migration of Quartz.NET 2.6.0 to 3.5.0

### DISCUSSION THREAD

**Discussion by elvir-isanovic** [quartznet/quartznet#1782] [Q&A]
Hi,

I'm using Quartz.NET for a longer period and we want now wo upgrade Quartz.NET to latest version. In production we have a lot of triggers and I wonder is it possible to do this migration smoothly and will it work with existing 2.6.0 DB schema?

Thanks,
Elvir

**Comment by lahma** [comment]
You should follow changes in each release notes (I've put an effort into them!). 

But indeed, there's a DB migration required from 2.6 to 3.x, and [it's here](https://github.com/quartznet/quartznet/blob/main/database/schema_26_to_30.sql), pretty simple one. Examples are always for SQL Server and you should test against non-production system / backup before jumping the gun.
